### ISSUE TITLE
Allow members to join the "leadership" mailing list

### ISSUE THREAD

**Issue by brandondiamond** [hackerunion-archives/hackerunion.org#29]
To be determined: allow readonly or read/write access.

This is toward the goal of establishing complete transparency.

**Comment by johndbritton** [comment]
I'm :+1: for the leadership list being publicly readonly. Show some commitment to the project and anyone on the list can add you if you ask.

Simple and lightweight.

**Comment by arelenglish** [comment]
I like that idea. The barrier to participate/ get involved should be low, but I like the idea that it's readonly until someone invites you. It's a good practice that has made services like Dribbble work well. I think it could work here.

**Comment by petergault** [comment]
Yeah I support this as well. It is important to get a variety of perspectives on what people are looking for in the hacker union.
